charts/trino/values.yaml

```
@@ -86,6 +86,9 @@ server:
     enabled: false
     maxReplicas: 5
     targetCPUUtilizationPercentage: 50
+    # server.autoscaling.targetCPUUtilizationPercentage -- Target CPU to scaling up and down. To not scale based on CPU, set to "".
+    targetMemoryUtilizationPercentage: 80
+    # server.autoscaling.targetMemoryUtilizationPercentage -- Target memory to scaling up and down. To not scale based on memory, set to "".
```

If you put the comments before the property, you don't have to use the fully qualified property name. I'd also suggest a slightly different phrasing, similar to [Kubernetes API docs](https://github.com/kubernetes/api/blob/master/autoscaling/v1/types.go#L55).
```suggestion
    # -- Target average CPU utilization, represented as a percentage of requested CPU. To disable scaling based on CPU, set to an empty string.
    targetCPUUtilizationPercentage: 50
    # -- Target average memory utilization, represented as a percentage of requested memory. To disable scaling based on memory, set to an empty string.
    targetMemoryUtilizationPercentage: 80
```